android/lib/src/main/java/com/marianhello/react/BackgroundGeolocationModule.java

```diff
@@ -433,8 +455,13 @@ private void sendEvent(ReactContext reactContext, String eventName, @Nullable Wr
     }
 
     public boolean hasPermissions() {
-        //TODO: implement
-        return true;
+        final Activity currentActivity = this.getCurrentActivity();
```

Comment:
I was laughing because I totally forgot the second step, not that it was a bad suggestion